tutorials/rag/agentic_rag.py

```diff
@@ -0,0 +1,271 @@
+"""
+This script demonstrates how to build an agentic RAG pipeline using Humanloop and OpenAI's API.
+
+The model can choose to call the retrieval tool, or instead ask for clarification from the user.
+
+This leverages tool calling: https://platform.openai.com/docs/guides/function-calling
+
+It assumes that Prompts and Tools are managed in the client's code and logging and evals are done via Humanloop.
```

Comment:
this script now doesn't demonstrate evals at all. (if so, this mention of evals should be separated. e.g. "This script assumes that ... and logging is done via Humanloop. To run evals on flows like the one built here, see ..."

Reply:
Will add an eval step at the end